src/plugins/intel_cpu/src/emitters/tpp/common/kernel_executors/brgemm.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "brgemm.hpp"
+
+#include "emitters/tpp/common/utils.hpp"
+#include "transformations/tpp/common/op/brgemm.hpp"
+
+#define PRINT(X) ss << #X << " = " << X << "\n"
+#define HASH(X)  seed = dnnl::impl::hash_combine(seed, X)
+
+namespace ov::intel_cpu::tpp {
+
+BrgemmKernelConfig::BrgemmKernelConfig(const element::Type& in0_dtype, const element::Type& in1_dtype)
+    : BrgemmGenericKernelConfig(),
+      m_static_params(std::make_shared<StaticParams>(in0_dtype, in1_dtype)) {}
+
+bool BrgemmKernelConfig::operator==(const BrgemmKernelConfig& rhs) const {
+    return BrgemmGenericKernelConfig::operator==(rhs) &&
+           (get_static_params() == rhs.get_static_params() ||
+            *get_static_params() == *(rhs.get_static_params()));
```

Comment:
Looks like in many methods `m_compile_flags` is missed. Could we add it to `operator==`, `compute_hash()`, `to_sring()`? Otherwise we will think that two `Brgemm` configs with the same all parameters except beta - are equal. But it's not right.

Reply:
`m_compile_flags` is the info combined from `m_static_params->m_compile_flags` and `beta`, both are in `operator==` and `compute_hash()`. I added `m_compile_flags ` to `to_sring()` to explicit show. What do you think?